integration_tests/mockApis/manageUsersApi.ts

```diff
@@ -50,17 +48,7 @@ const ping = () =>
   })
 
 export default {
-  stubManageUser: (
-    {
-      name,
-      roles,
-    }: {
-      name: string
-      roles: UserRole[]
-    } = {
-      name: 'john smith',
-      roles: [],
-    },
-  ): Promise<[Response, Response]> => Promise.all([stubUser(name), stubUserRoles(roles)]),
+  stubManageUser: stubUser,
   stubManageUsersPing: ping,
+  stubManageUserRoles: stubUserRoles,
```

Comment:
we don't actually use the Manage Users `getUserRoles` / `/users/me/roles` endpoint anywhere